### DISCUSSION TITLE
Profile 이미지 출력을 위한 이미지 경로를 별도 등록할 수 있나요?

### DISCUSSION THREAD

**Discussion by whatwant** [awesome-devblog/awesome-devblog#722] [Q&A]
눈코입 없는 이미지는 식상하고, 내 프로필 이미지가 나오면 좋겠는데

naver / medium / velog / github 등의 블로그 포스팅에는 프로필 이미지가 나오는 것 같고
tistory 등은 나오지 않고 ... 맞나요?

원하면 별도의 프로필 이미지 경로를 등록할 수도 있을까요?

**Comment by sarojaba** [comment]
답변이 너무 늦었네요.
현재로선 따로 등록하는 방법은 없습니다.
